src/test/ui/suggestions/appropriate-type-param-turbofish.stderr

```diff
@@ -0,0 +1,254 @@
+error[E0283]: type annotations needed
+  --> $DIR/appropriate-type-param-turbofish.rs:3:35
+   |
+LL |         vec![1, 2, 3].into_iter().collect();
+   |                                   ^^^^^^^ cannot infer type for type parameter `B` declared on the associated function `collect`
+   |
+   = note: cannot satisfy `_: FromIterator<i32>`
+help: consider specifying the type argument in the method call
+   |
+LL |         vec![1, 2, 3].into_iter().collect::<Vec<T>>();
```

Comment:
Only thing I would like to do is replace the type params in the suggestions with `_` to avoid "`T` doesn't exist" errors when applying the change.